src/sentry/models/organization.py

```diff
@@ -81,6 +81,7 @@ class Organization(Model):
 
     flags = BitField(flags=(
         ('allow_joinleave', 'Allow members to join and leave teams without requiring approval.'),
+        ('enhanced_privacy', 'Enable enhanced privacy controls to limit personally identifiable information (PII) as well as source code in things like emails.'),
```

Comment:
this might go beyond just notifications at some point, so I wanted to keep it semi open ended

(I dont have an example of what yet, as most things are good as long as they're over SSL)